openghg/store/_obssurface.py

```diff
@@ -405,82 +386,72 @@ def read_multisite_aqmesh(
         """
-        from collections import defaultdict
-
-        from openghg.standardise.surface import parse_aqmesh
-        from openghg.store import assign_data, datasource_lookup, load_metastore
-        from openghg.util import hash_file
-        from tqdm import tqdm
-
-        data_filepath = Path(data_filepath)
-        metadata_filepath = Path(metadata_filepath)
-
-        # Load the ObsSurface object for retrieve
-        obs = ObsSurface.load()
-        # Load the metadata store
-        metastore = load_metastore(key=obs._metakey)
-        # Get a dict of data and metadata
-        processed_data = parse_aqmesh(data_filepath=data_filepath, metadata_filepath=metadata_filepath)
-
-        results: resultsType = defaultdict(dict)
-        for site, site_data in tqdm(processed_data.items()):
-            metadata = site_data["metadata"]
-            measurement_data = site_data["data"]
-
-            file_hash = hash_file(filepath=data_filepath)
-
-            if obs.seen_hash(file_hash=file_hash) and overwrite is False:
-                raise ValueError(
-                    f"This file has been uploaded previously with the filename : {obs._file_hashes[file_hash]}."
-                )
-                break
-
-            combined = {site: {"data": measurement_data, "metadata": metadata}}
-
-            required_keys = (
-                "site",
-                "species",
-                "inlet",
-                "network",
-                "instrument",
-                "sampling_period",
-                "measurement_type",
-            )
-
-            lookup_results = datasource_lookup(
-                metastore=metastore, data=combined, required_keys=required_keys, min_keys=5
-            )
-
-            uuid = lookup_results[site]
-
-            # Jump through these hoops until we can rework the data assignment functionality to split it out
-            # into more sensible functions
-            # TODO - fix the assign data function to avoid this kind of hoop jumping
-            lookup_result = {site: uuid}
-
-            # Create Datasources, save them to the object store and get their UUIDs
-            data_type = "surface"
-            datasource_uuids = assign_data(
-                data_dict=combined,
-                lookup_results=lookup_result,
-                overwrite=overwrite,
-                data_type=data_type,
-            )
-
-            results[site] = datasource_uuids
-
-            # Record the Datasources we've created / appended to
-            obs.add_datasources(uuids=datasource_uuids, data=combined, metastore=metastore)
-
-            # Store the hash as the key for easy searching, store the filename as well for
-            # ease of checking by user
-            obs.set_hash(file_hash=file_hash, filename=data_filepath.name)
-
-        obs.save()
-        # Close the metadata store and write new records
-        metastore.close()
-
-        return results
+        raise NotImplementedError(
```

Comment:
So I guess we should make an explicit note that this is no longer supported? Maybe in the CHANGELOG.md?

Reply:
Added.